src/Game.js

```diff
@@ -0,0 +1,167 @@
+console.log("Towers of Hanoi!")
+
+var Game = function (){
+  this.gameBoard = new GameBoard();
+  this.firstSelection = -1;
+}
+
+Game.prototype.displayPegElements = function (pegElementsArray) {
+  var gameBoard = document.getElementById("game-board");
+  // Todo: Do I need this validation check? 
+  if (!Array.isArray(pegElementsArray)){
+    console.log("please provide an array")
+    return false
+  }
+  pegElementsArray.forEach((element) => {
+    gameBoard.appendChild(element);
+  });
+
+  this.initPegEventListeners();
+  return 
+};
+
+Game.prototype.clearPegElements = function() {
```

Comment:
This is not as secure as creating a class that has this elements inside. Using prototype to modify the defined object is a bit of a brute force hammer.